addons/l10n_ec/data/account_tax_report_data.xml

```diff
@@ -0,0 +1,3052 @@
+            <field name="sequence">1</field>
+        </record>
+
+        <record id="tax_report_line_104_525" model="account.tax.report.line">
+            <field name="name">Impuesto generado(525)</field>
+            <field name="code">c525</field>
+            <field name="tag_name">525 (Reporte 104)</field>
+            <field name="report_id" ref="tax_report_104"/>
+            <field name="parent_id" ref="tax_report_line_bee1b28d03f8b9ce12d26731523e991f"/>
+            <field name="sequence">1</field>
+        </record>
+        <record id="tax_report_line_793c8789e969442a99e8d4d98143daa5" model="account.tax.report.line">
+            <field name="name">IVA generado en la diferencia entre adquisiciones y notas de crédito con distinta tarifa (ajuste en positivo al crédito tributario)</field>
+            <field name="report_id" ref="tax_report_104"/>
+            <field name="parent_id" ref="tax_report_line_6b0babafa55bd198bf713446ad396300"/>
+            <field name="sequence">1</field>
+            <field name="formula">None</field>
+        </record>
+
+        <record id="tax_report_line_104_526" model="account.tax.report.line">
+            <field name="name">Impuesto generado(526)</field>
+            <field name="code">c526</field>
+            <field name="tag_name">526 (Reporte 104)</field>
+            <field name="report_id" ref="tax_report_104"/>
+            <field name="parent_id" ref="tax_report_line_793c8789e969442a99e8d4d98143daa5"/>
+            <field name="sequence">1</field>
+        </record>
+        <record id="tax_report_line_1ab31e68c903be53dd945c85d0ea5ddd" model="account.tax.report.line">
+            <field name="name">IVA generado en la diferencia entre adquisiciones y notas de crédito con distinta tarifa (ajuste en negativo al crédito tributario)</field>
+            <field name="report_id" ref="tax_report_104"/>
+            <field name="parent_id" ref="tax_report_line_6b0babafa55bd198bf713446ad396300"/>
+            <field name="sequence">1</field>
+            <field name="formula">None</field>
+        </record>
+
+        <record id="tax_report_line_104_527" model="account.tax.report.line">
+            <field name="name">Impuesto generado(527)</field>
+            <field name="code">c527</field>
+            <field name="tag_name">527 (Reporte 104)</field>
+            <field name="report_id" ref="tax_report_104"/>
+            <field name="parent_id" ref="tax_report_line_1ab31e68c903be53dd945c85d0ea5ddd"/>
+            <field name="sequence">1</field>
+        </record>
+        <record id="tax_report_line_19e7604d3e6e5fbe0d7e936a387e650e" model="account.tax.report.line">
+            <field name="name">Importaciones de bienes (incluye activos fijos) gravados tarifa 0%</field>
+            <field name="report_id" ref="tax_report_104"/>
+            <field name="parent_id" ref="tax_report_line_6b0babafa55bd198bf713446ad396300"/>
+            <field name="sequence">1</field>
```

Comment:
weird to have so many lines with sequence 1

Reply:
Resolved